manifest_staging/charts/gatekeeper/values.yaml

```diff
@@ -145,6 +145,7 @@ controllerManager:
   priorityClassName: system-cluster-critical
   disableCertRotation: false
   tlsMinVersion: 1.3
+  clientCAName: ""
```

Comment:
do we need to update the helm chart to set this value?

Reply:
Yip! Missed that. Updating chart as well